.github/workflows/pkgci_regression_test.yml

```
@@ -133,7 +133,7 @@ jobs:
           # CPU
           - name: cpu_llvm_task
             models-config-file: models_cpu_llvm_task.json
-            runs-on: nodai-amdgpu-w7900-x86-64
+            runs-on: persistent-cache
```

We could be a bit more descriptive here
```suggestion
            runs-on:
              - self-hosted # must come first
              - persistent-cache
              - Linux
              - X64
```

(planning ahead to having persistent Windows and/or arm64 runners)

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#choosing-self-hosted-runners